NEWS.md

```diff
@@ -200,6 +200,14 @@ Tooling Improvements
 
 * Printing of `MethodError` and methods (such as from `methods(my_func)`) is now prettified and colored consistently
   with printing of methods in stacktraces ([#45069]).
+  
+Regressions
+-----------
+
+* Precompilation takes significantly longer, 2x-3x compilations slowdowns are common
+* Some type-unstable code runs orders of magnitude slower ([#48612])
+* Inplace operations on high-dimensional array views may allocate ([#49332])
+* Compile cache gets excessively invalidated on clusters ([#48217])
```

Comment:
This is a bug that is being diagnosed currently. I don't think it's right to highlight this here (I can't speak to the other issues above)